index.js

```diff
@@ -285,26 +284,26 @@ exports.expressCreateServer = function (hook_name, args, callback) {
     // create data to hold comment reply information:
     let data;
     try {
-        data = JSON.parse(fields.data);
+      data = JSON.parse(fields.data);
     } catch (err) {
-      res.json({code: 1, message: "data must be a JSON", data: null});
+      res.json({code: 1, message: 'data must be a JSON', data: null});
       return;
     }
 
-    let replyIds, replies;
+    let replyIds; let replies;
```

Comment:
surprised you prefer this syntax

Reply:
I don't; thanks for pointing that out. Fixed.